app/src/main/java/org/wikipedia/suggestededits/SuggestedEditsContributionsFragment.kt

```diff
@@ -370,15 +329,56 @@ class SuggestedEditsContributionsFragment : Fragment(), SuggestedEditsTypeItem.C
         fun bindItem(contribution: Contribution) {
             view.setTitle(contribution.description)
             view.setDescription(contribution.title)
-            view.setImageUrl(contribution.imageUrl)
             view.setIcon(contribution.editType)
-            view.setPageViewCountText(contribution.pageViews)
             getPageView(view, contribution)
+            getImageDetails(view, contribution)
         }
 
-        private fun getPageView(view: SuggestedEditsContributionsItemView, contribution: Contribution) {
+        private fun getImageDetails(itemView: SuggestedEditsContributionsItemView, contribution: Contribution) {
             val disposables = CompositeDisposable()
```

Comment:
If you'd like to use `CompositeDisposable()`, does it mean that the `Observables` actions can be `dispose`?

Reply:
Do you mean it should be disposed? lmk if the latest update makes sense?

Reply:
Yes, it makes sense!